anvill/python/anvill/ida/idaprogram.py

```diff
@@ -48,6 +51,26 @@
 
 class IDAProgram(Program):
     def __init__(self, arch, os):
+        arg_parser = argparse.ArgumentParser()
```

Comment:
This should go into `get_program`, and be guarded to execute only once per program. Then the `get_program` function in both architectures should have an optional `base_address` kwarg.